Automattic/jetpack
# Contact Form: Removing code and renaming relevant references to ensure we use the package version

Fixes step 2 of this project: pf5801-Aj-p2

## Proposed changes:

* This PR addresses step 2 of this project: pf5801-Aj-p2. Specifically, it focusses on the following:
  * Deleting `module/contact-form` files.
  * Updating any references to deleted files, classes or functions to use the package version

Note - before merging, this diff needs to be deployed first: D147229-code

### Other information:


## Jetpack product discussion

pf5801-Aj-p2

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

* To test, apply this PR and make sure to test the contact form (block, also using the Classic Editor) on a test site and ensure that it continues to work as expected.
* Ensure no errors are being reported in logs.
